doctrine/multiple_entity_managers.rst

```diff
@@ -313,6 +313,6 @@ The same applies to repository calls::
             // ...
         }
 
-	You should now always fetch this repository using ``ManagerRegistry::getRepository()``.
+You should now always fetch this repository using ``ManagerRegistry::getRepository()``.
```

Comment:
Are you sure this does not belong to the `.. caution` directive and needs no be moved into the other direction? 🤔